include/librealsense2/dds/dds-device-server.cpp

```diff
@@ -136,12 +305,21 @@ void dds_device_server::init( const std::vector<std::string> &supported_streams_
 
     _publisher = DDS_API_CALL(_participant->create_publisher( PUBLISHER_QOS_DEFAULT, nullptr ));
 
+    // Create a notifications server
+    _dds_notifications_server = std::make_shared< dds_notifications_server >( _participant, _publisher, _topic_root );
+
+    // Create streams servers per each supporting stream of the device
     for( auto stream_name : supported_streams_names )
     {
         _stream_name_to_server[stream_name] = std::make_shared< dds_stream_server >( _participant,
-                                                                                    _publisher,
-                                                                                    _topic_root,
-                                                                                    stream_name );
+                                                                                     _publisher,
+                                                                                     _topic_root,
+                                                                                     stream_name );
     }
 }
 
+void dds_device_server::publish_notifications( const topics::raw::device::notifications& notifications_msg, bool new_reader_notification )
```

Comment:
I find myself confused about `new_reader_notification` and the `latched` argument that it's called inside.

Can you explain?

Reply:
I don't see any call to `publish_notifications` with this argument `false`... even though that's the default.